integration/presets/astro.ts

```diff
@@ -20,6 +20,9 @@ const astroNode = applicationConfig()
   .addDependency('@clerk/types', clerkTypesLocal)
   .addDependency('@clerk/localizations', clerkLocalizationLocal);
 
+const astroStatic = astroNode.clone().setName('astro-static').useTemplate(templates['astro-static']);
```

Comment:
Do we need to add an e2e template with hybrid and `prerender = true` ?

Reply:
That's a good to have but I think if we do hybrid test we can have both prerendered pages and just remove static e2e

Reply:
Added hybrid test